services/provider/interfaces/interfaces.go

```diff
@@ -6,6 +6,8 @@ type StorageClientStatus interface {
 	// no client is using it
 	GetPlatformVersion() string
 	GetOperatorVersion() string
+	GetClusterId() string
+	GetStorageClientNamespacedName() string
```

Comment:
I don't think we need to report this information on the status message as it is always constant and we are already decided to report these in the onboard message